sql/V1_12__votes.sql

```diff
@@ -0,0 +1,39 @@
+ALTER TABLE IF EXISTS proposals
+DROP CONSTRAINT IF EXISTS proposals_proposal_id_ux;
+
+ALTER TABLE IF EXISTS proposals
+ADD CONSTRAINT proposals_proposal_id_ux UNIQUE (chain_num, proposal_id);
```

Comment:
i needed to add this unique constraint on the proposals table because i wanted to utilize that column combination as a foreign key in the votes table. this provides at least two benefits, data integrity, in the sense that we won't add votes to the database for non-existent proposals, and also it provides a relationship that can be conveniently traversed in a postgraphile graphql query.

i.e. return all proposals and include the votes for each proposal:

```graphql
{
  allProposals(orderBy: PROPOSAL_ID_DESC, condition: {chainNum: 84}) {
    nodes {
      txHash
      proposalId
      votesByChainNumAndProposalId {
        nodes {
          voter
          option
        }
      }
    }
  }
}
```